packages/@aws-cdk/aws-events-targets/lib/log-group.ts

```diff
@@ -0,0 +1,59 @@
+import * as events from '@aws-cdk/aws-events';
+import * as iam from '@aws-cdk/aws-iam';
+import * as logs from '@aws-cdk/aws-logs';
+import * as cdk from '@aws-cdk/core';
+import { LogGroupResourcePolicy } from './log-group-resource-policy';
+/**
+ * Customize the CloudWatch LogGroup Event Target
+ */
+export interface LogGroupProps {
```

Comment:
can a role also be supplied? - some of our other event targets accept a role / create a singleton role for the target.

Reply:
It is not possible to pass a role when using a LogGroup as a target. I tried it initially and got an CloudFormation error back.

Reply:
gotcha, thanks for clarifying